add-on/src/lib/options.js

```diff
@@ -62,15 +62,21 @@ function buildDefaultIpfsNodeConfig () {
     // Until we have MulticastDNS+DNS, peer discovery is done over ws-star
     config.config.Addresses.Swarm = ['/dns4/ws-star1.par.dwebops.pub/tcp/443/wss/p2p-websocket-star']
     // Until DHT and p2p transport are ready, delegate + preload
+    // Note: we use .preload.ipfs.io and .delegate.ipfs.io as means of http sharding (12 instead of 6 concurrent requests)
     const delegates = [
-      '/dns4/node0.preload.ipfs.io/tcp/443/https',
-      '/dns4/node1.preload.ipfs.io/tcp/443/https'
+      '/dns4/node1.delegate.ipfs.io/tcp/443/https',
+      '/dns4/node0.delegate.ipfs.io/tcp/443/https'
     ]
     // Delegated Content and Peer Routing: https://github.com/ipfs/js-ipfs/pull/2195
-    // TODO: delegated routing blocked by https://github.com/libp2p/js-libp2p-delegated-content-routing/issues/12
-    // config.config.Addresses.Delegates = delegates
-    // TODO: are preloads needed? should Brave have own nodes?
-    config.preload = { enabled: true, addresses: delegates }
+    config.config.Addresses.Delegates = delegates
+    // TODO: when we have p2p transport, are preloads still needed? should Brave have own nodes?
+    config.preload = {
+      enabled: true,
+      addresses: [
+        '/dns4/node1.preload.ipfs.io/tcp/443/https',
```

Comment:
We will reuse `delegates` here, but for now I use `.preload.` hostnames to make it easier to see which requests are from libp2p's routing modules, and which ones are triggered by preload logic in js-ipfs.